### ISSUE TITLE
Support Czech font

### ISSUE THREAD

**Issue by corvik** [PaulWoitaschek/Voice#1660]
### Overview

Apk miss support czech names.  
From app:
![Screenshot_2023-01-25-21-33-53-292_de ph1b audiobook](https://user-images.githubusercontent.com/64494842/214685626-45794556-bf23-47b7-8adb-1ebb75af52c2.jpg)
![Screenshot_2023-01-25-21-33-49-725_de ph1b audiobook](https://user-images.githubusercontent.com/64494842/214685633-4dc55a7c-11f5-44fc-9017-b04b5fd8178e.jpg)
![Screenshot_2023-01-25-21-33-39-970_de ph1b audiobook](https://user-images.githubusercontent.com/64494842/214685638-56fd4a8a-ea56-4dc5-a087-38bb2654d565.jpg)
 Another app without issue:
![Screenshot_2023-01-25-21-37-36-994-edit_ak alizandro smartaudiobookplayer](https://user-images.githubusercontent.com/64494842/214685746-f0b57644-f48e-4c1f-a326-c91584ac46f2.jpg)

### How to reproduce

Open app, use czech audiobooks.

### Version

Google play

### Android Version

12

### Device Model

Xiaomi 11t pro

**Comment by PaulWoitaschek** [comment]
Could you send me such a file where it's happening to my mail?

**Comment by PaulWoitaschek** [comment]
(Closing until the file arrives due to cleanups, will reopen later)

**Comment by corvik** [comment]
emaik sent,  btw now i have another phone and its same (Samsung A52s)
Thanks

**Comment by PaulWoitaschek** [comment]
Thanks for the report. You've sent me a `mp3` file.

Th bytes look like this:

```
54414C420000000E00000056CC8E20564C418A544F564B595452434B
```

 * `54414C42` = TALB -> the code for the name of the album
* `0000000E` = The size, 14
* `flags` = 0000` = encoding, utf8
* `56CC8E20564C418A544F564B595452434B` = content

That content now in utf8 translates to: `V̎ VLA�TOVKYTRCK`

which is what we see in the App.

Now if we try an encoding that's different from what the file says, i.e.
windows 1252 we see:
`VÌŽ VLAŠTOVKYTRCK`

TLDR: File has a broken encoding, the tag editor has a bug.
